db/schema.rb

```diff
@@ -753,7 +762,7 @@
 
   create_table "wmap_requests", :force => true do |t|
     t.string   "host"
-    t.string   "address"
+    t.string   "address",    :limit => nil
```

Comment:
This might cause issues. We've since this `:limit => nil` flicker on and off before.  Did we ever figure out the problem @shuckins-r7?